test/rules/assertions/checkValues.js

```diff
@@ -206,6 +225,44 @@ export default {
         },
       ],
     },
+    {
+      code: `
+      /**
+       * @variation
+       */
+      function quux (foo) {
+
+      }
+      `,
+      errors: [
+        {
+          line: 3,
+          message: 'Missing JSDoc @variation.',
```

Comment:
Should be "Missing value for JSDoc variation."

Reply:
Thanks, fixed, along with the messages for other Missing tag values that preexisted.